Gemini/src/Migrations/Version20180530031926.php

```diff
@@ -12,20 +12,37 @@
 {
     public function up(Schema $schema)
     {
-        // this up() migration is auto-generated, please modify it to your needs
-        $this->abortIf(
-            $this->connection->getDatabasePlatform()->getName() !== 'mysql',
-            'Migration can only be executed safely on \'mysql\'.'
-        );
+      $this->addSql(
```

Comment:
just a heads up here. I agree that at this stage of the project dropping is totally Ok! But, we should avoid dropping if the table is there and full of tiny little URIs.

Reply:
Agreed. Future version files should do some sort of ETL process. I just thought that it would be way more complex than necessary at this stage. It would be easier for the few people who need it right now to do an external ETL process than to code it here.

Reply:
I agree! So,  probably for a next step (new issue, future) of this module, to add an ETL process console scrip that is not a migration? I mean, something that can read via drupal's rest API all URI, check back to Fedora and is able to reconstruct (in the way Fedora 3 did) the mapping/index? Would be good to have, there are just too many ways a mapper can fail and too many ways a mapper could need to be reconstructed. We will need to support @Natkeeran that has some kind of claw hybrid running in the wild.

Reply:
Once this gets release we'll def. be on the hook for migrating.  At least now we can do it less manually :+1: